modules/core/server/views/partials/header.server.view.html

```diff
@@ -196,4 +196,8 @@
   </div>
   <!-- /Header for authenticated users -->
 
+  <div class="container">
+    <language-switch></language-switch>
+  </div>
```

Comment:
Shall we have this at the beginning behind feature flag and make it visible once the experience is more pleasant? Now there won't really be anything to look at and it can be a bit confusing.